bogo/src/main.rs

```diff
@@ -230,6 +226,50 @@ impl Options {
     }
 }
 
+#[derive(Debug, Default)]
+struct Credentials {
+    default: Credential,
+    additional: Vec<Credential>,
+}
+
+impl Credentials {
+    fn most_recent_mut(&mut self) -> &mut Credential {
```

Comment:
Nit: maybe just call this `last_mut()`? There's no real "recency" involved anyway...